axelor-base/src/main/resources/domains/Partner.xml

```diff
@@ -110,6 +110,9 @@
     <many-to-one name="mainAddress" ref="com.axelor.apps.base.db.Address"
       title="Address"/>
 
+    <many-to-one name="businessCountry" ref="com.axelor.apps.base.db.Country"
+      title="Business Country"/>
```

Comment:
Update the Partners demo data and set a businessCountry on them, just put France on all the Partners